app.json

```diff
@@ -14,5 +14,19 @@
       "plan": "heroku-postgresql",
       "as": "DATABASE"
     }
-  ]
+  ],
+  "env": {
+    "SECRET_STORE_BASE": {
+      "required": true,
+      "generator": "secret"
```

Comment:
pretty cool if its sticky

Reply:
Perhaps take these couple out of the guide then if done automatically. Other stuff in here isn't mentioned either in the same vein.

Reply:
The guide and the app.json unfortunately cover separate processes. This config is used if you use the "quick-deploy" buttons but if you want to create an app from the group up with the CLI, it doesn't apply this configuration from the repo.

Reply:
I stand corrected; we could re-use the app.json to create apps with the CLI, but it’s via a Heroku CLI addon:
```
heroku plugins:install api
heroku api POST /apps --body '{"name": "example-app"}'
```

Reply:
not sure why quick deploy wouldn't suffice for all?